Logger/FileLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+   public class FileLogger : BaseLogger
+    {
+        private string _FilePath { get; }
+        public FileLogger(string filePath) { _FilePath = filePath; }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            if(!string.IsNullOrEmpty(_FilePath))
+            {
+                File.AppendAllText(_FilePath, $"{DateTime.Now}{ClassName}{logLevel + ":"} {message}\n");
```

Comment:
```suggestion
                File.AppendAllText(_FilePath, $"{DateTime.Now}{ClassName}{logLevel}: {message}\n");
```
Don't combine string interpolation and string concatenation unless you are trying to split your interpolation onto multiple lines.